scraping/pdf_module/pdf_scraper/pdf_xml_converter.py

```diff
@@ -190,7 +190,7 @@ def print_xml(sections: list[(str, str)], output_filepath: str, document_creatio
 
     Args:
         sections (list[str, str)]): List of section text tuples in form of (header_text, paragraph_text).
-        output_filepath (str): Filepath of the file to be written in form of "*.xml", where * is a wildcard.
+        output_filepath (str): Filepath of the file to be written in form of `*.xml`, where * is a wildcard.
```

Comment:
why are you in this file and why single quotes?
```suggestion
        output_filepath (str): Filepath of the file to be written in form of "*.xml", where * is a wildcard.
```

Reply:
Because * is a special symbol for Sphinx so it messes it up. using ` instead of " fixes it.